### PR TITLE
Christian   save scores

### PR DESCRIPTION
**Pull request by clyngbo**
yay - done

### CODE REVIEW

File: app/Http/Controllers/Api/RoomController.php [hunk 1]
```diff
@@ -20,11 +23,12 @@ public function index()
         $rooms = Room::get();
 
         foreach ($rooms as $room) {
+            $score = DB::table('scores')->where('room_id', '=', $room->internal_id)->latest()->first();
```
Comment (AngelOD) [inline app/Http/Controllers/Api/RoomController.php]: Replace
DB::table('scores')->where('room_id', '=', $room->internal_id)
with
$room->scores()

File: app/Http/Controllers/Api/RoomController.php [hunk 2]
```diff
@@ -42,12 +46,12 @@ public function show($roomID)
         $room = Room::where('internal_id', '=', strtoupper($roomID))->first();
 
         if (empty($room)) { return response()->json('Invalid parameter data', 400); }
-
+        $score = DB::table('scores')->where('room_id', '=', $room->internal_id)->latest()->first();
```
Comment (AngelOD) [inline app/Http/Controllers/Api/RoomController.php]: Here as well

**Review by AngelOD — CHANGES_REQUESTED** [review]
Other than those two changes (and subsequent testing) it looks good.

File: app/Http/Controllers/Api/RoomController.php (hunk 1 above)
Comment (clyngbo) [reply app/Http/Controllers/Api/RoomController.php]: done

File: app/Http/Controllers/Api/RoomController.php (hunk 2 above)
Comment (clyngbo) [reply app/Http/Controllers/Api/RoomController.php]: done

**Review by AngelOD — APPROVED** [review]

**Pull request merged**